extension-style/api/metalava.txt

```diff
@@ -2,16 +2,18 @@
 package com.mapbox.maps.extension.style {
 
   public final class StyleExtensionImpl implements com.mapbox.maps.extension.style.StyleContract.StyleExtension {
-    method public java.util.List<com.mapbox.maps.extension.style.StyleContract.StyleImageExtension> getImages();
-    method public java.util.List<kotlin.Pair<com.mapbox.maps.extension.style.layers.Layer,com.mapbox.maps.LayerPosition>> getLayers();
+    method public error.NonExistentClass! getImages();
+    method public error.NonExistentClass! getLayers();
```

Comment:
Are we adding `error.NonExistentClass` thing in metalava 🙈 . I guess its needed to pass the checks, correct ?

Reply:
I simply did run the script and got following output. Already reverted that, we need to track separately why metalava did break